blockchain/src/bitgo.ts

```diff
@@ -0,0 +1,57 @@
+import { BitGo, Wallet } from 'bitgo';
+import bitcore from "zcash-bitcore-lib";
+import env from './env';
+import log from './log';
+import { getNetwork } from './node';
+
+let bitgoWallet: Wallet;
+
+export async function initBitGo() {
+  if (!env.BITGO_ACCESS_TOKEN || !env.BITGO_WALLET_ID) {
+    log.info('BITGO environment variables not set, nooping initBitGo');
+    return;
+  }
+
+  // Assert that we're on mainnet
+  const network = getNetwork();
+  if (network !== bitcore.Networks.mainnet) {
+    throw new Error(`BitGo cannot be used on anything but mainnet, connected node is ${network}`);
+  }
+
+  const proxy = env.FIXIE_URL || undefined;
+  const bitgo = new BitGo({
+    env: 'prod', // Non-prod ZEC is not supported
+    accessToken: env.BITGO_ACCESS_TOKEN,
+    proxy,
+  });
+  bitgoWallet = await bitgo.coin('zec').wallets().get({ id: env.BITGO_WALLET_ID });
+  log.info(`Initialized BitGo wallet "${bitgoWallet.label()}"`);
+  if (proxy) {
+    log.info(`Proxying BitGo requests through ${proxy}`);
+  }
+}
+
+export async function getContributionAddress(id: number) {
+  if (!bitgoWallet) {
+    throw new Error('Must run initBitGo before getContributionAddress');
+  }
+
+  // Attempt to fetch first
+  const label = `Contribution #${id}`;
```

Comment:
Just a thought. Would it make sense to add any additional data to the label? Version? Date? Might come in handy at some point.

Reply:
Given that most of the data could be corroborated with the source of truth (the contribution) I don't feel strongly about this, but it would be easy to add.

I think if there are any major changes to getting the address that knowing the version would be helpful, we'd probably want to spin up a new wallet, but that's just a guess.